src/brain_teasers/python/k_closest_points/README.md

```diff
@@ -0,0 +1,27 @@
+# K Closest Points to (0, 0)
+
+This python function finds the K closest points to (0, 0) in a given list of points.
+
+```
```

Comment:
This looks nice, and you can add `python` after the backtick  it will make the documented code more readable
```python
# your code
```

Reply:
Added : )